src/app/tests/TestAclAttribute.cpp

```diff
@@ -166,12 +147,12 @@ void TestAclAttribute::TestACLDeniedAttribute(nlTestSuite * apSuite, void * apCo
                                    chip::app::ReadClient::InteractionType::Subscribe);
 
         chip::app::AttributePathParams attributePathParams[2];
-        attributePathParams[0].mEndpointId  = kTestEndpointId;
-        attributePathParams[0].mClusterId   = kTestDeniedClusterId1;
+        attributePathParams[0].mEndpointId  = chip::Test::kTestEndpointId;
+        attributePathParams[0].mClusterId   = chip::Test::kTestDeniedClusterId1;
```

Comment:
Why alll the extra `chip::`?  This applies to a bunch of places here.

Reply:
ok will review them all

Reply:
because the pigweed namespace is also called `Test`... putting only Test chooses `pw::unit_test::internal::Test`